### ISSUE TITLE
data.js is written in JSON style but it have JS syntax and file extension

### ISSUE THREAD

**Issue by Grawl** [artem-malko/tars-jade#1]
[`data.js`](https://github.com/tars/tars-jade/blob/6e175f7afdec0829a5d450968240c5ed970063b5/markup/modules/head/data/data.js) located in `/markup/modules/head/data/` is written in wrong way. It is JS at all? Looks like JSON. My IDE fills every string as error here.

It must be written as a real JS:

``` js
data={
    head: {
        defaults: {
            title: 'default title',
            useSocialMetaTags: true
        }
    }
}
```

or as proper JSON:

``` json
{
    "head": {
       "defaults": {
            "title": "default title",
            "useSocialMetaTags": true
        }
    }
}
```

**Comment by artem-malko** [comment]
Resolved in TARS #176

**Comment by Grawl** [comment]
Good.
